.github/workflows/test.yml

```diff
@@ -28,18 +74,27 @@ jobs:
       with:
         python-version: ${{ matrix.cfg.py }}
 
+    - name: Download the wheel
+      uses: actions/download-artifact@v4
+      with:
+        name: wheel
+        path: dist
+
+    - name: Install the wheel for Windows configuration
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        $wheel = Get-ChildItem -Path dist -Filter "periodictable*.whl" | Select-Object -First 1
+        python -m pip install $wheel.FullName
+      shell: pwsh
```

Comment:
Not liking the duplication.

Presumably the following fails in Windows?
```yaml
name: Install the wheel
      run: python -m pip install dist/periodictable*.whl
```
How about:
```yaml
name: Install the wheel
      run: |
        cd dist
        python -m pip install periodictable*.whl
```
Does windows restore the working directory after powershell exits, or does the cd leak into the next command?